tests/testthat/test-zzz-plumb_api.R

```diff
@@ -57,6 +57,37 @@ test_that("errors are thrown", {
   expect_error(available_apis("crayon"), "No Plumber APIs found for package")
 })
 
+test_that("edit opens correct file", {
+  # Redefine editor so that file.edit doesn't try to open a file
+  orig_opts <- options(editor = function(name, file, title) {
+    cat(file, " test file attempted to open\n")
```

Comment:
There should be a function called `cat0()`

```suggestion
    cat(file, " test file attempted to open\n", sep = "")
```